ReactTask/src/views/DoctorProfile.js

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect } from 'react';
+import { BoxFilter, BoxSearch } from './../components/Boxes';
+
+import {
+  StylBoxBtnPatients,
+  StylBoxPatientContent,
+  StylBoxFlexColumn,
+  StylBoxDoctor,
+  StylBoxPatients,
+  StylBoxRowPadding,
+  StylBoxRow,
+  StylSubTitlePosition,
+  StylTitlePatients,
+  StylTitleName,
+  AvatarActive,
+  BtnPatients,
+  CardPatientProfileDoctor,
+} from '../components/';
+
+import avatar1 from '../img/avatars/avatar-doctor.png';
```

Comment:
Better if u put all these patient avatars into ur `dbProfileDoctor.json` and map them in ur component `CardPatientProfileDoctor` 
````
<BoxPatientsList>
      {dataPatients.map((item) => (
        <StylBoxCardPatient key={item.id}>
          <StylBoxRowLine>
            <StylBoxDoctor>
              <StylBoxImgPatient
                src={item.image}  <----- here u just bring this avatar from db.json property!
                alt={item.alt}
              />
              <StylBoxColumnInfo>
                <StylTitleCard>{item.name}</StylTitleCard>
                <StylSubTitleCard status={item.status}>
                  {item.statusText}
                </StylSubTitleCard>
              </StylBoxColumnInfo>
            </StylBoxDoctor>
            <BtnMore />
          </StylBoxRowLine>
          <BoxDataPatient data={item.data} />
          <BoxInfoPatient info={item.info} imgWay={scheduleImg} />
        </StylBoxCardPatient>
      ))}
    </BoxPatientsList>
````

Reply:
Fixed